Client/src/Components/Profile/PermissionsSection/PermissionsTab.tsx

```diff
@@ -0,0 +1,76 @@
+import * as Yup from 'yup'
+
+import { Box, Button, Grid, Typography } from '@material-ui/core'
+import { FastField, Form, Formik } from 'formik'
+import { fetchAllAdmins, updatePermissions } from '../../../Remote/Endpoints/PermissionsEndpoint'
+import { AdminList } from './AdminList'
+
+import { MuiTextFieldFormik } from '../../Forms/FormikFields'
+import React, { useEffect } from 'react'
+import { classStyles } from '../../../appTheme'
+import { useState } from 'react'
+
+export default function PermissionsTab() {
+
+  const validationSchema = Yup.object().shape({
+    email: Yup.string()
+      .email().required("Email Address")
+  })
+
+  const [adminListState, setAdminListState] = useState([])
+
+  const handleAddAdmin = async (newAdmin: any) => {
+    await updatePermissions({
+      email: newAdmin.email,
+      operation: "add"
+    })
+    const adminListState = await fetchAllAdmins()
+    setAdminListState(adminListState)
+  }
+
+  const handleRemoveAdmin = async (newAdmin: string) => {
```

Comment:
its very optional but you could have handleUpdatePermissions(updatePermissionInfo)
await updatePermissions(updatePermissionInfo)

and have both callers build the IUpdatePermissionInfo object, they would provide the operation type and the email. this will remove the duplication